src/igvfd/schemas/auxiliary_set.json

```diff
@@ -65,6 +65,16 @@
                 "quantification DNA barcode sequencing",
                 "variant sequencing"
             ],
+            "enum_descriptions": {
```

Comment:
I think these would be clearly if you indicate WHY cell hashing, lipid conjugation, etc is used (I think to extend the barcoding space, but not 100%)

Reply:
added purposes